page01_index.php

```diff
@@ -1,177 +1,171 @@
-      <script>
-        var slideIndex = 0;
-        carousel();
+						<div class="mySlides w3-third">
+							<img class="w3-round-large" src="images/placeholder.jpg" alt="Cherries" width="210" height="110" style="width:100%;max-width:230px">
+							<h3>Album name</h3>
+						</div>
 
-        function carousel() {
-          var i;
-          var x = document.getElementsByClassName("mySlides");
-          for (i = 0; i < x.length; i++) {
-            x[i].style.display = "none";
-          }
-          slideIndex++;
-          if (slideIndex > x.length) {slideIndex = 1}
-          x[slideIndex-1].style.display = "block";
-          setTimeout(carousel, 2000); // Change image every 2 seconds
-        }
-      </script>
+					</div>			
+				</div>
 
 
+				<script>
+					var slideIndex = 0;
+					carousel();
 
-    </div>
-  </div>
+					function carousel() {
+						var i;
+						var x = document.getElementsByClassName("mySlides");
+						for (i = 0; i < x.length; i++) {
+							x[i].style.display = "none";
+						}
+						slideIndex++;
+						if (slideIndex > x.length) {slideIndex = 1}
+						x[slideIndex-1].style.display = "block";
+						setTimeout(carousel, 2000); // Change image every 2 seconds
+					}
+				</script>
 
+				<br>
+				<br>
 
-  <br>
-  <br>
+				<!-- Playlist photos / Queries -->
+				<!-- Responsive Grid. Four columns on tablets, laptops and desktops. Will stack on mobile devices/small screens (100% width) -->
 
+				<div class="w3-row-padding w3-center">
+					<div class="w3-col m3">
+						<a href="Playlist_Genre.rock .php"> <img src="images/placeholder.jpg" class="w3-button w3-round w3-opacity w3-hover-opacity-off" style="width:100%"  alt="ROCK"> </a>
+					</div>
 
 
-  <!-- Playlist photos / Queries -->
-  <!-- Responsive Grid. Four columns on tablets, laptops and desktops. Will stack on mobile devices/small screens (100% width) -->
-  <div class="w3-row-padding w3-center">
+					<div class="w3-col m3">
+						<a href="Playlist_Genre.pop.php"> <img src="images/placeholder.jpg" class="w3-button w3-round w3-opacity w3-hover-opacity-off" style="width:100%"  alt="POP"> </a>
+					</div>
 
-    <div class="w3-col m3">
-      <a href="Playlist_Genre.rock .php"> <img src="images/placeholder.jpg" style="width:100%" class="w3-hover-opacity" alt="The mist over the mountains"> </a>
-    </div>
 
-    <div class="w3-col m3">
-      <a href="playlist_genre.php"> <img src="images/placeholder.jpg" style="width:100%" class="w3-hover-opacity" alt="The mist over the mountains"> </a>
-    </div>
 
-    <div class="w3-col m3">
-      <a href="playlist_genre.php"> <img src="images/placeholder.jpg" style="width:100%" class="w3-hover-opacity" alt="The mist over the mountains"> </a>
-    </div>
+					<div class="w3-col m3">
+						<a href="Playlist_Genre.country.php"> <img src="images/placeholder.jpg" class="w3-button w3-round w3-opacity w3-hover-opacity-off" style="width:100%"  alt="COUNTRY"> </a>
```

Comment:
added href links within the images so users can click them and be taken to the playlist page directly